src/backends/perf_pt/mod.rs

```diff
@@ -276,8 +276,8 @@ impl Tracer for PerfPTTracer {
         }
 
         // At the time of writing, we have to use a fresh Perf file descriptor to ensure traces
-        // start with a `PSB+` packet sequence. This is required for correct decoding. For this
-        // reason we have to re-initialise for each new tracing session.
+        // start with a `PSB+` packet sequence. This is required for correct instruction- and
```

Comment:
On second reading I know why the "-" is at the end of "instruction" -- but it's pretty distracting. I think it's safe enough to just drop it.

Reply:
We even have enough space on the line to repeat `-level`. Looks good?